editor/src/core/shared/uid-utils.ts

```diff
@@ -20,12 +20,15 @@ import { objectMap } from './object-utils'
 
 export const UtopiaIDPropertyPath = PP.create(['data-uid'])
 
+const GeneratedUIDs: Set<string> = new Set()
+
 export function generateUID(existingIDs: Array<string>): string {
   const fullUid = UUID().replace(/\-/g, '')
   // trying to find a new 3 character substring from the full uid
   for (let i = 0; i < fullUid.length - 3; i++) {
     const id = fullUid.substring(i, i + 3)
-    if (!existingIDs.includes(id)) {
+    if (!existingIDs.includes(id) && !GeneratedUIDs.has(id)) {
```

Comment:
If we're using the `GeneratedUIDs` we don't need to use `existingIDs`

Reply:
my assumption was that existingIDs might come from a loaded file, not generated in this session